build/common.ps1

```diff
@@ -245,7 +254,9 @@ Function Install-DotnetCLI {
 
     # Install the 2.x runtime because our tests target netcoreapp2x
     Trace-Log "$DotNetInstall -Runtime dotnet -Channel 2.2 -i $CLIRoot -NoPath"
-    & $DotNetInstall -Runtime dotnet -Channel 2.2 -i $CLIRoot -NoPath
+    # Work around the following install script bug https://github.com/dotnet/install-scripts/issues/152.
+    # Start a new process to avoid the ev getting populated.
+    & powershell $DotNetInstall -Runtime dotnet -Channel 2.2 -i $CLIRoot -NoPath
```

Comment:
The tests build against .NET `2.1` runtime and it's LTS. So, why are we using `2.2` that went out of support (_on 23-12-2019_) more than year ago?

Reply:
This doesn't affect the building of the tests and the runtime the tests run against.
It affects the integration tests that we run. 2.2 covers the widest 2.x scenarios because of the roll forward behavior.